### PR TITLE
Add net.zdechov.app.C_evo

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
> A clone of Civilization II.
- [X] The domain used for the application ID is [controlled by the application developer(s)][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor to the project. If not, I contacted upstream developers about this submission. **Link:** http://c-evo.org/bb/viewtopic.php?f=4&t=229&p=1869#p1869

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build net.zdechov.c_evo

File: net.zdechov.c_evo.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  - name: c-evo
+    buildsystem: simple
+    build-commands:
+      - |
+        . /usr/lib/sdk/freepascal/enable.sh
+        cd tags/1.3.6/
+        lazbuild Integrated.lpr
+    post-install:
+      - mkdir -p $FLATPAK_DEST/share/c-evo
+      - cp -r tags/1.3.6/AI $FLATPAK_DEST/share/c-evo
```
Comment (bbhtt) [inline net.zdechov.c_evo.yaml:28]: 1.3.6 is going to break on every new tag

**Review by bbhtt — CHANGES_REQUESTED** [review]
Metadata - icon, metainfo should be upstream.

File: net.zdechov.c_evo.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      - cp tags/1.3.6/c-evo $FLATPAK_DEST/share/c-evo
+      - install -Dm644 tags/1.3.6/Install/deb/c-evo.desktop $FLATPAK_DEST/share/applications/c-evo.desktop
+      - install -Dm644 icon.png $FLATPAK_DEST/share/icons/hicolor/128x128/apps/c-evo.png
+      - install -Dm644 tags/1.3.6/Install/deb/c-evo.xml $FLATPAK_DEST/share/mime/application/c-evo.xml
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml $FLATPAK_DEST/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - mkdir -p $FLATPAK_DEST/bin/
+      - install -Dm755 start.sh $FLATPAK_DEST/bin/c-evo
+    sources:
+      - type: svn
+        url: https://svn.zdechov.net/c-evo
```
Comment (hfiguiere) [inline net.zdechov.c_evo.yaml:49]: According to the manual

>               URL of the svn repository, including branch/tag part

So here you should add the tags path.

File: net.zdechov.c_evo.yaml (hunk 1 above)
Comment (hfiguiere) [reply net.zdechov.c_evo.yaml:28]: see also https://github.com/flathub/flathub/pull/5833/files#r1836745009

File: net.zdechov.c_evo.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        revision: 628
+      - type: file
+        path: icon.png
+      - type: file
+        path: net.zdechov.c_evo.metainfo.xml
+      - type: script
+        commands:
+          - #!/bin/sh
```
Comment (hfiguiere) [inline net.zdechov.c_evo.yaml:57]:
```suggestion
```

File: net.zdechov.c_evo.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
+app-id: net.zdechov.c_evo
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
```
Comment (hfiguiere) [inline net.zdechov.c_evo.yaml:3]: 24.08

Comment (hfiguiere) [reply net.zdechov.c_evo.yaml:3]: oh wait maybe freepascal needs to exist for 24.08 first.

File: net.zdechov.c_evo.yaml (hunk 2 above)
Comment (hfiguiere) [reply net.zdechov.c_evo.yaml:49]:
```suggestion
        url: https://svn.zdechov.net/c-evo/tags/1.3.6
```

Comment (hfiguiere) [reply net.zdechov.c_evo.yaml:49]: and remove it from the build commands.

File: net.zdechov.c_evo.yaml (hunk 1 above)
Comment (hfiguiere) [reply net.zdechov.c_evo.yaml:28]: this could also have been `subdir` but this should be necessary with what I referenced above.

**Comment by Mailaender** [comment]
bot, build net.zdechov.c_evo

File: net.zdechov.app.C_evo.yaml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+modules:
+  - shared-modules/gtk2/gtk2.json
+
+  - name: c-evo
+    buildsystem: simple
+    build-commands:
+      - |
+        . /usr/lib/sdk/freepascal/enable.sh
+        lazbuild Integrated.lpr
+    post-install:
```
Comment (hfiguiere) [inline net.zdechov.app.C_evo.yaml:26]:
```suggestion
```

File: net.zdechov.app.C_evo.yaml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+      - cp -r Sounds $FLATPAK_DEST/share/c-evo
+      - cp -r Tribes $FLATPAK_DEST/share/c-evo
+      - cp Fonts.txt $FLATPAK_DEST/share/c-evo
+      - cp Language.txt $FLATPAK_DEST/share/c-evo
+      - cp Language2.txt $FLATPAK_DEST/share/c-evo
+      - cp c-evo $FLATPAK_DEST/share/c-evo
+      - install -Dm644 Install/flatpak/net.zdechov.app.C-evo.desktop $FLATPAK_DEST/share/applications/net.zdechov.app.C-evo.desktop
+      - install -Dm644 icon.png $FLATPAK_DEST/share/icons/hicolor/128x128/apps/net.zdechov.app.C-evo.png
+      - install -Dm644 Install/flatpak/net.zdechov.app.C-evo.appdata.xml $FLATPAK_DEST/share/metainfo/net.zdechov.app.C-evo.appdata.xml
+      - mkdir -p $FLATPAK_DEST/bin/
```
Comment (hfiguiere) [inline net.zdechov.app.C_evo.yaml:44]:
```suggestion
```

**Pull request closed without merging**